R/Viewer.R

```diff
@@ -38,7 +39,6 @@ NULL
 #'
 #'   Viewer(tbl, tbl2)
 #' }
-#' @export
```

Comment:
Is this a standard? Because I prefer to see the name of the doc file and if it is exported over the function name instead. This renders reading code more effective than looking for export and name through possibly long docs

Reply:
ok - I will restore the reordering changes made here